src/app/services/carbon-estimation-util.service.ts

```diff
@@ -0,0 +1,76 @@
+import { Injectable } from '@angular/core';
+import { NumberObject, sumValues } from '../utils/number-object';
+import { startCase } from 'lodash-es';
+import { SVG } from '../carbon-estimation/carbon-estimation.constants';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class CarbonEstimationUtilService {
+  constructor() {}
+
+  public getOverallPercentageLabel = (emissions: NumberObject): string => {
+    const percentage = sumValues(emissions);
+    return this.getPercentageLabel(percentage);
+  };
+
+  public getPercentageLabel = (percentage: number): string => (percentage < 1 ? '<1%' : Math.round(percentage) + '%');
```

Comment:
Not introduced by this change but noticed this is a dupe of `tooltipFormatter` from `carbon-estimation.constants` (or vice versa). The treemap usage of `tooltipFormatter` could switch to using this from the service but I'm not quite sure what would be best to do with the custom tooltip and data label (unless this service provides them too?). It does seem good that these are tested in isolation.

Reply:
I can change this to be 
`public getPercentageLabel = tooltipFormatter;`
Then only one occurrence of the function and it will be covered by util service unit tests.